__fixtures__/test-project/package.json

```diff
@@ -21,5 +21,8 @@
   "prisma": {
     "seed": "yarn rw exec seed"
   },
-  "packageManager": "yarn@3.2.0"
-}
+  "packageManager": "yarn@3.2.0",
+  "scripts": {
+    "postinstall": ""
+  }
```

Comment:
Hmm, we don't want these in the fixture (didn't catch it before). I'll work on a code mod to clean it up at the very end.

Reply:
Ah, just saw this in the build steps below:
```
            scripts: {
               postinstall: '', // We do this so we don't have project:copy in the fixture
             },
```

Understood.

I think we do need to remove `"packageManager": "yarn@3.2.0",` Correct @jtoar ? I'm actually not sure where this is coming from in the first place...

Note: we will be adding this back in sooner than later once the CRWA template is stable on Yarn 3. My guess is that it's happening at one of these steps and we're not removing it then:
1. Create new project (from framework using Yarn3): https://github.com/redwoodjs/redwood/blob/main/tasks/test-project/test-project#L176-L186
2. Set Yarn 1 step: https://github.com/redwoodjs/redwood/blob/e7343b57a955ba748e813eb6dcfc8718e2b55a7c/tasks/test-project/test-project#L215-L223

So possible to remove it along with step 2 task? If there, we'd also not miss it when bumping everything to yarn 3.

Reply:
**Updates**
I confirmed this is happening before the project yarn install step.

I've now:
- removed this from existing fixture in `main` here https://github.com/redwoodjs/redwood/commit/0084bb981549b8311e35f8a2270da522887a634c
- added code to delete packageManger property here (and rebuilt fixture to confirm ok): https://github.com/redwoodjs/redwood/pull/4566/commits/023b2f9e523d2dcadc1b626c14a37aea3ca312d9